config/static/nodes/cgroup-v2/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- machine-config-cgroup-v2-master.yaml
```

Comment:
FYI, for our current use case we only need to enable cgroupv2 on the worker nodes.
Maybe in the future, if we have "IDM in control plane", we will need cgroupv2 on master nodes too.

Not an objection, just want to promote comprehensive understanding :)

Reply:
Understood. I will add additional notes into the README.md file for this. Thanks :)